scripts/everware-server

```diff
@@ -17,13 +17,37 @@ class Everware(app.JupyterHub):
 
     def init_handlers(self):
         super().init_handlers()
+        self.replace_handlers(handlers_map)
+
+    def replace_handlers(self, handlers_map):
+        """Replace default handler classes with new ones.
+
+        Parameters
+        ----------
+        handlers_map : dict
+            Contains a mapping old_handler_class:new_handler_class.
+        """
         for i, cur_handler in enumerate(self.handlers):
             new_handler = handlers_map.get(cur_handler[1])
             if new_handler:
```

Comment:
add warning in case `new_handler` is not there? or should it call `add_handler` instead? or should it be single method like `update_handler`?